src/main/java/org/bdgenomics/convert/ga4gh/BdgenomicsAlignmentRecordToGa4ghReadAlignment.java

```diff
@@ -0,0 +1,177 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bdgenomics.convert.ga4gh;
+
+import javax.annotation.concurrent.Immutable;
+
+import ga4gh.Common;
+import ga4gh.Reads;
+import org.bdgenomics.convert.AbstractConverter;
+import org.bdgenomics.convert.Converter;
+import org.bdgenomics.convert.ConversionException;
+import org.bdgenomics.convert.ConversionStringency;
+
+import org.bdgenomics.formats.avro.AlignmentRecord;
+import htsjdk.samtools.Cigar;
+import htsjdk.samtools.CigarOperator;
+import htsjdk.samtools.TextCigarCodec;
+import htsjdk.samtools.ValidationStringency;
+import htsjdk.samtools.CigarElement;
+
+
+
+import org.slf4j.Logger;
+
+import java.util.*;
+
+/**
+ * Convert bdg-formats AlignmentRecord to GA4GH ReadAlignment.
+ */
+@Immutable
+final class BdgenomicsAlignmentRecordToGa4ghReadAlignment extends AbstractConverter<org.bdgenomics.formats.avro.AlignmentRecord, ga4gh.Reads.ReadAlignment> {
```

Comment:
You can use imports above and simple class names here, i.e. `AbstractConverter<AlignmentRecord, ReadAlignment>`.  I've had to use fully qualified classnames in other converters where the source and target class have the same name.  Likewise the classname could be `AlignmentRecordToReadAlignment`.

Reply:
I kind of like the `Bdg` and `Ga4gh` prefixes for readability in the class name.  But will change it if you feel strongly?

Reply:
Nope, don't feel strongly, just less typing.  One of the benefits of the Guice stuff is that the long class names aren't visible to clients of the library.

Reply:
fixed.  There is still a place elsewhere where I use the fully class name for 
`ga4gh.Reads.ReadAlignment` as I like it for readability there, but if that's undesirable let me know